lib/sampling_strategy.ex

```diff
@@ -0,0 +1,7 @@
+defmodule Spandex.SamplingStrategy do
+  @moduledoc """
+  A Sampling strategy is used by spandex to calculate any sampling information for a new trace.
+  """
+
+  @callback calculate_sampling(trace_id :: Spandex.id(), opts :: Keyword.t()) :: any()
```

Comment:
Sometimes `sampling` is `any()`, sometimes it's a `map()` - if it's an opaque value set by `spandex_datadog` then it should be `any()`.